includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -914,6 +952,7 @@ private static function settings_appearance() {
                 'name'    => _x( 'Min Image File Size (KB)', 'settings', 'business-directory-plugin' ),
                 'default' => '0',
                 'group'   => 'images/general',
+				'class'   => 'wpbdp-half wpbdp-input-wide',
```

Comment:
Can these be changed to wpbdp-col-grid-2? Is wpbdp-input-wide needed?

Reply:
The `wpbdp-half` aligns the current container into a grid of 50% by 50% . The `wpbdp-col-grid-2` aligns child items in this order. It wont work in this layout